### PR TITLE
News: GRASS GIS 8.2.1 released

### PR DESCRIPTION
**Pull request by neteler**
- draft text, please check

TODO: add screenshot

### CODE REVIEW

File: content/news/2023_01_21_GRASS_GIS_8_2_1_released.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+graphical user interface has also seen improvements, with issues being
+resolved for the database manager, the image group dialog, the import
+of PostGIS and SQLite geometry data, the Postscript-based mapping tool,
+and the Single-Window GUI toolbar. Additionally, a correction has been added
+to the Jupyter notebooks for bounding box reprojection for `InteractiveMap`
+and the `GridModule` now cleans up temporary mapsets when an exception occurs.
+Concerning the documentation, all addons now register their keywords in
+the main keywords index, leading to a better overview of available
+functionality. For mobile devices, viewport support and hamburger menu-style
+table of content has been added. The PROJ library support received
```
Comment (veroandreo) [inline content/news/2023_01_21_GRASS_GIS_8_2_1_released.md:25]:
```suggestion
table of content have been added. The PROJ library support received
```

File: content/news/2023_01_21_GRASS_GIS_8_2_1_released.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+compared to the previous release, 8.2.0. The update includes a range of
+enhancements to the software's modules and graphical user interface.
+Some notable module improvements include various bug fixes for `r.kappa`,
+the addition of support for unregistering maps from other mapsets in
+`t.unregister`, and a fix for cross-validation in `v.surf.rst`. The
+graphical user interface has also seen improvements, with issues being
+resolved for the database manager, the image group dialog, the import
+of PostGIS and SQLite geometry data, the Postscript-based mapping tool,
+and the Single-Window GUI toolbar. Additionally, a correction has been added
+to the Jupyter notebooks for bounding box reprojection for `InteractiveMap`
```
Comment (veroandreo) [inline content/news/2023_01_21_GRASS_GIS_8_2_1_released.md:20]:
```suggestion
to the grass.jupyter for bounding box reprojection within the `InteractiveMap` class
```

File: content/news/2023_01_21_GRASS_GIS_8_2_1_released.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Concerning the documentation, all addons now register their keywords in
+the main keywords index, leading to a better overview of available
+functionality. For mobile devices, viewport support and hamburger menu-style
+table of content has been added. The PROJ library support received
+important fixes for area bounding boxes and longitude estimation.
+Compatibility fixes for PDAL 2.5.0 have been added, and updates have
+been made to the available docker images, especially the base images
+(Ubuntu and Debian). The Windows NSIS installer was also updated to fix
+rare installation problems. Overall, this release offers a more stable
+and efficient version of GRASS GIS for users.
```
Comment (veroandreo) [inline content/news/2023_01_21_GRASS_GIS_8_2_1_released.md:31]:
```suggestion
rare installation problems. Overall, this release offers a **more stable
and efficient version of GRASS GIS for users**.
```

**Review by veroandreo — APPROVED** [review]
I made this screenshot the other day, would you like to use it?

**Comment by neteler** [comment]
> I made this screenshot the other day, would you like to use it?

Yes, looks nice!

**Comment by neteler** [comment]
Looks fine to me:

image

**Comment by neteler** [comment]
Importantly, I have also incremented the GRASS GIS version to 8.2.1 in  `data/grass.json`.

**Comment by neteler** [comment]
Sigh, forgot to update
https://github.com/OSGeo/grass-website/blob/master/content/about/history/releases.md

**Pull request merged**